src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
```

Comment:
move on all values to constants and move on them outside the function:
example
```
  const SHORT_TERM = 3;
  const SHORT_TERM_DISCOUNT = 20;
```

Reply:
Not fixed. Create variables for all 'magic numbers' (40, 7, 3)